content/Reducing_Power_Consumption.md

```diff
@@ -26,30 +27,30 @@ When waiting for a vblank event, this would be a BAD example:
 ```
 
 A better example would be a procedure as shown below. In this case the
-vblank interrupt must be enabled, and your vblank interrupt procedure
+v-blank interrupt must be enabled, and your v-blank interrupt handler
 must set vblank_flag to a non-zero value.
 
 ```
   ld   hl,vblank_flag  ;hl=pointer to vblank_flag
   xor  a               ;a=0
  @@wait:               ;wait...
   halt                 ;suspend CPU - wait for ANY interrupt
-  cp   a,(hl)          ;vblank flag still zero?
-  jr   z,@@wait        ;wait more if zero
+  cp   a,(hl)          ;vblank_flag still zero?
+  jr   z,@@wait        ;keep waiting if zero
   ld   (hl),a          ;set vblank_flag back to zero
 ```
```

Comment:
Markdown normally requires a blank line between code blocks and the next paragraph.

Reply:
Fixed